ERCS/eip-xxxx.md

```diff
@@ -0,0 +1,266 @@
+---
```

Comment:
```suggestion
---
eip: 7601
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.